santa/src/title.rs

```diff
@@ -28,7 +28,7 @@ impl TitleScreen {
                     .into_widget(ctx)
                     .container()
                     .padding(16)
-                    .bg(app.cs.fade_map_dark)
+                    .bg(Color::BLACK.alpha(0.8))
```

Comment:
Some of the Santa title screen was sometimes illegible if the map happens to be zoomed in. I made the black background more opaque.
![Screenshot from 2021-11-04 14-27-39](https://user-images.githubusercontent.com/1664407/140422627-53c5ccb2-a1fe-407d-89a5-a86ad57c2404.png)
When we happen to be zoomed out, looks a bit off, but eh.